.github/CODEOWNERS

```diff
@@ -77,11 +77,12 @@ subprojects/docs/src/docs/userguide/reference/directory_layout.adoc
 subprojects/docs/src/docs/userguide/troubleshooting/version_catalog_problems.adoc @gradle/bt-jvm
 
 # Dev-prod team
-.teamcity/                                  @gradle/bt-developer-productivity
-.github/                                    @gradle/bt-developer-productivity
-gradle/shared-with-buildSrc/                @gradle/bt-developer-productivity
-build-logic/                                @gradle/bt-developer-productivity
-build-logic-commons/                        @gradle/bt-developer-productivity
-build-logic-settings/                       @gradle/bt-developer-productivity
-subprojects/*/build.gradle*                 @gradle/bt-developer-productivity
-*.gradle.kts                                @gradle/bt-developer-productivity
+.teamcity/                   @gradle/bt-developer-productivity
+.github/                     @gradle/bt-developer-productivity
+gradle/shared-with-buildSrc/ @gradle/bt-developer-productivity
+build-logic/                 @gradle/bt-developer-productivity
+build-logic-commons/         @gradle/bt-developer-productivity
+build-logic-settings/        @gradle/bt-developer-productivity
+subprojects/*/build.gradle*  @gradle/bt-developer-productivity
```

Comment:
I thought this `subprojects/*/build.gradle*` include would be the problem. Though maybe we need to use `/*.gradle.kts` to only match the root build/settings gradle.

Reply:
Found the probable answer in the [CODEOWNERS docs](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners)

```
# The `docs/*` pattern will match files like
# `docs/getting-started.md` but not further nested files like
# `docs/build-app/troubleshooting.md`.
docs/*  docs@example.com
```

The interesting bits when looking at the docs:
 -  The `docs/*` pattern will match files like `docs/getting-started.md` but not further nested files like `docs/build-app/troubleshooting.md`.
 - _Fun fact, which I didn't know about_: Order is important; the last matching pattern takes the most precedence.